src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1111,14 +1111,17 @@ private void InitializePackagesFolderInfo(string packagesFolderPath, bool exclud
 
                 var installedPackagesInDependencyOrder = await GetInstalledPackagesInDependencyOrder(nuGetProject, token);
 
+                var isDependencyBehaviorIgnore = resolutionContext.DependencyBehavior == DependencyBehavior.Ignore;
```

Comment:
Actually, I take that back. Passing down a bool is better in this case since the bool is used in multiple places.